api/matlab/arrus/mexcuda/iqRaw2Lri.cu

```diff
@@ -359,6 +364,17 @@ void mexFunction(int nlhs, mxArray * plhs[],
 //                                                                       dev_centX     + iPart*nTxPerPart, 
 //                                                                       minRxTang, maxRxTang, fs, fn, sos, initDel, 
 //                                                                       nZPix, nXPix, nSamp, nElem, nTxInThisPart);
+//         iqRaw2Lri<<<blocksPerGrid, threadsPerBlock, sharedPerBlock>>>(dev_iqLri + iPart*nZPix*nXPix*nTxPerPart, 
```

Comment:
Why keep this comment? (and others)

Reply:
One obsolete call to the kernel is removed. The other stays as it will be used with the kernel version optimized for linear arrays.